### DISCUSSION TITLE
Randomize Reasearch List

### DISCUSSION THREAD

**Discussion by Corwin-dev** [Die4Ever/rollercoaster-tycoon-randomizer#27] [Ideas]
I think randomizing the research order at the start of the scenario could add some nice complexity.

But I love your plugin and I really appreciate all the work you've put into it.

Cheers

**Comment by Die4Ever** [comment]
Unfortunately this isn't possible for a plugin to do in OpenRCT2 yet, but I'm always watching when they add more plugin features (or maybe one day I'll do it myself). Thanks for the feedback!
